run.sh

```diff
@@ -0,0 +1,111 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+#   sh run.sh lint path/to/config.jsonc path/to/package
```

Comment:
This is recommending `sh`, but towards the end of the file, this script is executing `bash`. Would it make sense to recommend people run this script in `bash` for consistency? I don't know shell scripting well, so this is not a leading question, just wondering.

Reply:
It works on either sh or bash. However, sh is available on almost (if not all) *nix environments, while bash is not available on some minimal OS images. Not a big preference, this could be changed since it's probably safe to assume bash is available.